components/Prompts.js

```diff
@@ -26,8 +28,8 @@ const Prompts = () => {
         <div className="main-content p-3 justify-between flex flex-col relative text-center shadow-sm">
           <div className="p-2">
             { image ?
-              <p>{ polyglot.t('image_true', { name: user.screenname }) }</p>:
-              <p>{ polyglot.t('image_false', { name: user.screename })  }</p>
+              <p>{ __(polyglot.t('image_true', { name: user.screenname })) }</p>:
```

Comment:
No - what I want is `<p>{ __('image_true', { name: user.screenname }) }</p>`
And then in the __ function, use polyglot.